system-parachains/asset-hubs/asset-hub-polkadot/src/xcm_config.rs

```diff
@@ -587,6 +588,7 @@ pub mod bridging {
 			);
 
 			pub const KusamaNetwork: NetworkId = NetworkId::Kusama;
+			pub KusamaEcosystem: Location = Location::new(2, [GlobalConsensus(KusamaNetwork::get())]);
```

Comment:
`KsmLocation` is the KSM asset id/location, while `KusamaEcosystem` is the location of the Kusama ecosystem root/relay.
The values are the same/equal, but conceptually the two variables define separate things with separate purposes. 

I merged them because in practice we'll never change the id/location of KSM/DOT and even if we did we'd catch this in tests and create separate `KusamaEcosystem` then.